src/Responder/ChainedResponder.php

```diff
@@ -51,19 +51,16 @@ public function __invoke(
     /**
      * @inheritDoc
      *
-     * @throws InvalidArgumentException If a responder does not implement the correct interface.
+     * @throws ResponderException
+     *  If $classes does not implement the correct interface.
      */
-    protected function assertValid(array $data)
```

Comment:
These were called `$data` to match the `Set` parameter name. But that's an okay change.